src/transport_ofi.h

```diff
@@ -40,6 +40,10 @@
 #define ENABLE_TARGET_CNTR 0
 #endif
 
+#if DISABLE_NONFETCH_AMO
+#define SOS_OFI_CXI_QUIET_RATE 200
+#endif
```

Comment:
Should we keep a comment here regarding the reason of tis?

Reply:
Sure - added.